tests/e2e/support/objects/app-files/resource/webOffice.ts

```diff
@@ -21,11 +23,14 @@ export const removeCollaboraWelcomeModal = async (page: Page): Promise<void> =>
   })
   if (!versionSet) {
     await editorMainFrame.locator(collaboraWelcomeModal).waitFor()
-    await page.keyboard.press('Escape')
+    const welcomeModal = editorMainFrame.frameLocator(collaboraWelcomeModal)
+    await welcomeModal.locator(collaboraWelcomeSlide).last().click()
```

Comment:
isn't there other way rather than using .last()

Reply:
I see `.last()` as the best option to select the last slide.